Src/EventCounterCollector/EventCounterCollector.Tests/EventCounterCollector.Tests/EventCounterCollectorDiagnoticListener.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Tracing;
+using System.Text;
+
+namespace EventCounterCollector.Tests
+{
+    internal class EventCounterCollectorDiagnoticListener : EventListener
+    {
+        public IList<string> EventsReceived { get; private set; }
+        public EventCounterCollectorDiagnoticListener()
+        {
+            this.EventsReceived = new List<string>();
```

Comment:
make EventsReceived some kind of concurrent collection (e.g. queue). not sure if it is important right away, but it has proven needed/useful in other cases and can save a bunch of time debugging broken tests